docs/translations/api-docs/grid/grid.json

```diff
@@ -61,76 +58,6 @@
       "description": "Styles applied to {{nodeName}} if {{conditions}}.",
       "nodeName": "the root element",
       "conditions": "<code>wrap=\"reverse\"</code>"
-    },
-    "align-items-xs-center": {
```

Comment:
Technically these are breaking changes, but I am not sure if they are worth documenting.

Reply:
We could anyway add a migration guide similar to this one - https://github.com/mui-org/material-ui/pull/24496/files#diff-c66b437e5d952a0998e624e32687b4f3114020624f2fee56d4894f27579874f3R1205

Reply:
Any breaking change is worth documenting.

Reply:
Just added it :)